client/state/sites/constants.js

```diff
@@ -44,6 +44,7 @@ export const SITE_REQUEST_OPTIONS = [
 	'page_on_front',
 	'page_for_posts',
 	'podcasting_archive',
+	'podcasting_category_id',
```

Comment:
This is one place in the rest of Calypso that is querying the podcasting options...